examples/using-medium/gatsby-config.js

```diff
@@ -6,7 +6,12 @@ module.exports = {
     {
       resolve: `gatsby-source-medium`,
       options: {
+        // To test the limit parameter
+        // Replace the publication username with a user
+        // Limit is optional - set quite low as example user doesn't have that many posts
         username: `smartive`,
+        // username: `@ReactEurope`,
+        // limit: 20, // note: the graphql query in src/pages/index.js also imposes a limit
```

Comment:
commeting this one out is intentional?

Reply:
I would think that just documenting it in plugin docs would be enough, but we can keep it here